core-metastore/src/main/java/org/zalando/nakadi/repository/db/SubscriptionDbRepository.java

```diff
@@ -155,6 +162,33 @@ public void deleteSubscription(final String id)
         }
     }
 
+    public static class OffsetLimit {
```

Comment:
I see multiple problems with this class.  But I wonder if we can sidestep all of it by actually deprecating use of `offset` parameter in the API first? https://nakadi.io/manual.html#/subscriptions_get*offset